tensorflow/lite/micro/arena_allocator/recording_single_arena_buffer_allocator.h

```diff
@@ -26,14 +26,28 @@ namespace tflite {
 // allocation usage.
 class RecordingSingleArenaBufferAllocator : public SingleArenaBufferAllocator {
  public:
+  // TODO(vamsimanchala): Will be removed with http://b/246776144
```

Comment:
let's change the TODO to refer to the bug instead of a username. See http://go/tott/502 fpr more context/

Reply:
Done.